validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/ConfigSequences.java

```diff
@@ -13,7 +14,11 @@
 import com.google.daq.mqtt.sequencer.SequenceRunner;
 import com.google.daq.mqtt.util.JsonUtil;
 import java.util.Date;
+import java.util.HashMap;
 import org.junit.Test;
+import udmi.schema.BlobBlobsetConfig;
```

Comment:
extra stuff not necessary anymore?

Reply:
What?

Reply:
All the changes in the file... aren't they leftover from when you had the test itself in there?

Reply:
Oh haha. Yes of course. One moment